css/properties/image-orientation.json

```
@@ -30,10 +30,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "13.1"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "13.1"
```

```suggestion
              "version_added": "13.4"
```

See the following:
https://github.com/mdn/browser-compat-data/blob/0df9f6042832a7e6f4dd236589155b9c923ab7d4/browsers/safari_ios.json#L147-L152